### PR TITLE
Pom 30 send emails Early Allocation Part III

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

File: app/controllers/early_allocations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  def load_prisoner
+    @offender = OffenderService.get_offender(params[:prisoner_id])
+    @allocation = AllocationVersion.find_by!(offender_id_from_url)
+    @pom = PrisonOffenderManagerService.get_pom(@prison, @allocation.primary_pom_nomis_id)
+  end
+
+  def save_model_and_email
+    @early_assignment.save!
+
+    PomMailer.send_early_allocation_email(email: @offender.ldu.email_address,
```
Comment (rossjones) [inline app/controllers/early_allocations_controller.rb]: Do we have a clear steer on what is expected if there is no LDU email address, or do we not even get this far?

File: app/controllers/early_allocations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                                          prisoner_name: @offender.full_name,
+                                          prisoner_number: @offender.offender_no,
+                                          pom_name: @allocation.primary_pom_name,
+                                          pom_email: 'x',
```
Comment (rossjones) [inline app/controllers/early_allocations_controller.rb]: 'x'

File: app/controllers/early_allocations_controller.rb (hunk 1 above)
Comment (starswan) [reply app/controllers/early_allocations_controller.rb]: Yes the story now includes a dead-end page if we don't have an LDU email address

**Comment by rossjones** [comment]
Have told codeclimate to be quiet, just down 0.07% coverage

**Pull request merged**